libs/coin-framework/src/account/helpers.test.ts

```diff
@@ -241,40 +185,13 @@ describe(getAccountSpendableBalance.name, () => {
     });
   });
 
-  describe("given a ChildAccount", () => {
-    it("should return the account balance", () => {
-      const sampleAccountBalance = new BigNumber(10);
-      childAccount.balance = sampleAccountBalance;
-      expect(getAccountSpendableBalance(childAccount)).toEqual(sampleAccountBalance);
-    });
-  });
-
   describe("given a TokenAccount", () => {
     it("should return the token account spendable balance", () => {
       const sampleAccountBalance = new BigNumber(10);
       tokenAccount.spendableBalance = sampleAccountBalance;
       expect(getAccountSpendableBalance(tokenAccount)).toEqual(sampleAccountBalance);
     });
   });
-
-  describe("given an unknown type Account", () => {
```

Comment:
I don't understand why you remove those tests. They are limited to `ChildAccount` (it is not obvious)?

Reply:
because there is no longer an unknown type. it only can be `TokenAccount`. This test was covering the case where user data was having a wrong data, which shouldn't happen. in the new implementation we will ignore it from the import.